frontend/src/components/Forum/mockData.tsx

```diff
@@ -0,0 +1,55 @@
+import {ForumThread, User} from "../../api";
+
+const headers = ['Tytuł tematu', 'Liczba odpowiedzi', 'Założyciel', 'Data założenia','Akcje'];
```

Comment:
Czy to na pewno powinno być w mockData? Nie są to dane które będziemy kiedyś brać z backendu tylko stałe.